docs/source/cuda/device-management.rst

```diff
@@ -34,6 +34,7 @@ Users can then create a new context with another device.
 
 
 .. function:: numba.cuda.select_device(device_id)
+   :noindex:
```

Comment:
It seems like all the "noindex" entries are already documented in the reference. Does it make sense to have duplicate docs for them?